hedyorg/hedy
# 💻 teacher mode for not logged in users

Fixes #5183 

**How to test?**
- you should be logged out
- click on start teaching in the landing page
- test the teacher mode.

> We need to add a `PREVIEW_TEACHER_MODE_PASSWORD` env. variable.
